NitroxClient/GameLogic/ItemContainers.cs

```diff
@@ -26,7 +27,7 @@ public ItemContainers(IPacketSender packetSender, EntityMetadataManager entityMe
             this.entityMetadataManager = entityMetadataManager;
         }
 
-        public void BroadcastItemAdd(Pickupable pickupable, Transform containerTransform)
+        public void BroadcastItemAddAndSendMetadata(Pickupable pickupable, Transform containerTransform)
```

Comment:
Also with this changes the value only gets updated when adding the tank to the inventory and not when we use breath the air in it or rise to the surface. Maybe we need some kind of system to send this data repentantly (like every 1-3 seconds or so).